governance.md

```diff
@@ -1,14 +1,88 @@
-# Tekton Governance
+# Tekton Bootstrap Governance
 
-The Tekton project was donated to the [CDF](cd.foundation) in March 2019.
-Establishing governance is still a work in progress and for now, Tekton is
-overseen by the Knative Build WG.
+The initial bootstrap committee will consist of 5 individuals and one governance facilitator who are core stakeholders and/or contributors.
 
-## Procedures
+Members are:
 
-### Creating a new repository
+* [Andrew Bayer](https://github.com/abayer) (CloudBees)
+* [Vincent Demeester](https://github.com/vdemeester) (RedHat)
+* [Christie Wilson](https://github.com/bobcatfish) (Google)
+* [Dan Lorenc](https://github.com/dlorenc) (Google)
+* [Kim Lewandowski](https://github.com/kimsterv) (Google)
 
-New repositories can be created in the `tektoncd` Github organization with approval from two
-maintainers (for now these can be found in [Github](https://github.com/orgs/tektoncd/teams/maintainers/members).
+The committee MUST:
+
+* Represent a cross-section of interests, not just one company
+* Balance technical, architectural, and governance expertise since its initial mission is the establishment of structure around contributions, community, and decision-making
+* Hold staggered terms, sufficient to ensure an orderly transition of power via elections as designed and implemented by the committee (see below for specific deliverables)
+* Provide designated alternates in cases where quorum is required but not attainable with the current set of members
+* Communicate with the Continuous Delivery Foundation on a regular cadence
+
+## Governance Facilitator Role (optional)
+
+The governance facilitator role is a non-voting, non-technical advisory position that helps ensure the bootstrapping process is being adhered to, that all steering committee meetings are inclusive, and key milestones toward governance are met.
+The facilitator is a role, meaning it may be occupied by a single individual, or a series of individuals over time.
+The goal of this position is servant leadership for the project, community, and stakeholders.
+The committee may choose to make this a permanent role at their discretion.
+Deliverables may include creation of mailing lists, project tracking boards, governance draft documents, and so forth.
+
+## Committee Deliverables
+
+The committee will be responsible for a series of specific artifacts and activities as outlined below.
+
+### Initial Charter
+
+This document will define how the committee is to manage the project until it has transitioned to an elected steering body, as well as what governance must be in place.
+The Kubernetes Steering Committee Charter Draft serves as a good example. 
+
+A charter should cover all of the following topics:
+
+* Scope of rights and responsibilities explicitly held by the committee
+* Committee structure that meets the requirements above
+* Election process, including: 
+  * special elections in the case someone resigns or is impeached
```

Comment:
in all seriousness i cant stop myself from asking: how does a person get impeached O.O

Reply:
This is a list of things this steering committee is required to outline/determine. So, we get to decide!